### PR TITLE
Add integrated content links

### PR DESCRIPTION
**Pull request by hursey013**
Resolves https://github.com/18F/samhsa-prototype/issues/365
...with the exception of finding a home for `All facilities are licensed by their states, and provide assessments`.

### CODE REVIEW

File: src/components/Results/ResultsWarning.js [hunk 1]
```diff
@@ -3,28 +3,33 @@ import tw from 'tailwind.macro';
...
-        css={tw`underline text-gray-darkest`}
-      >
-        Learn more about different types of treatment
-      </Link>
-      . All facilities are licensed by their states, and provide assessments.
-    </p>
+    <ul css={tw`pl-4 ml-4 list-outside list-disc`}>
+      <li css={tw`mb-2`}>
+        Call a facility before you visit to make sure they have the services you
+        need. What to expect when you call.{' '}
```
Comment (allisonnorman) [inline src/components/Results/ResultsWarning.js]: What to expect when you call is duplicated. I think we should just leave the hyperlinked sentence.

File: src/components/Results/ResultsWarning.js [hunk 2]
```diff
@@ -3,28 +3,33 @@ import tw from 'tailwind.macro';
...
+        </Link>
+        .
+      </li>
+      <li>
+        Not sure what you need?{' '}
+        <Link
+          to="/content/treatment-options/types-of-treatment"
+          css={tw`underline text-gray-darkest`}
+        >
+          Learn more about different types of treatment
```
Comment (allisonnorman) [inline src/components/Results/ResultsWarning.js]: @amandacostello - I think this needs a 'the' - Learn more about THE different types of treatment.

File: src/components/Results/ResultsWarning.js [hunk 3]
```diff
@@ -3,28 +3,33 @@ import tw from 'tailwind.macro';
 import 'styled-components/macro';
 import { Link } from 'react-router-dom';
 
-import Warning from './ResultsWarning';
+import Warning from '../Warning';
 
 const ResultsWarning = () => (
   <Warning heading="Before you call">
```
Comment (allisonnorman) [inline src/components/Results/ResultsWarning.js]: @amandacostello - I think we talked about switching this up. It probably should be "before you visit" or "before you go" as the warning is more around calling before you visit.

File: src/components/Results/ResultsWarning.js (hunk 2 above)
Comment (allisonnorman) [reply src/components/Results/ResultsWarning.js]: Or, "learn more about treatment options"

File: src/components/Results/ResultsWarning.js (hunk 1 above)
Comment (hursey013) [reply src/components/Results/ResultsWarning.js]: Fixed 👍

**Comment by allisonnorman** [comment]
Made the content updates and resolved the change request. Ready to merge, @hursey013

**Comment by hursey013** [comment]
@allisonnorman did we want to find a spot for `All facilities are licensed by their states, and provide assessments.`?

**Pull request merged**